### DISCUSSION TITLE
Please don't add a text editor

### DISCUSSION THREAD

**Discussion by michael-odonovan** [jscad/OpenJSCAD.org#1155] [General]
Just agreeing with your reluctance.
If you use neovim or vscode or sim, why would you want another clunky option?
Less work for you.

**Comment by hrgdavor** [comment]
Not sure, what you are referring to here :)

**Comment by z3dev** [comment]
@michael-odonovan welcome!

we try to provide tools for our users, and focus on the WEB based UI and CLI. But there are those that want more  powerful tools, with debuggers, etc. 

I would prefer to maintain a plug-in to Vscode. What do you think?

**Comment by z3dev** [comment]
Well... it's time to move forward... how about collaborative editing?

https://firepad.io/

https://etherpad.org/
